test/test_sync.py

```diff
@@ -326,3 +309,51 @@ def test_sync_failures(mc):
     # check synced images
     assert not os.path.exists(os.path.join(driver_dir, "img1.png"))
     assert os.path.exists(os.path.join(driver_dir, "images", "img2.jpg"))
+
+def test_multiple_tables(mc):
+    project_name = "mediasync_test"
```

Comment:
I suggest to add `_multiple` to project name so we do now overwrite another test